tensorflow/contrib/tpu/profiler/pip_package/cloud_tpu_profiler/main.py

```diff
@@ -47,6 +54,8 @@ def main(unused_argv=None):
   cmd.append('--logdir='+logdir)
```

Comment:
The three flags are used here. I modified the tf.flags to absl.flags for all the 5 flags, while only 2 are newly added.